bayesian/api_v1.py

```diff
@@ -403,82 +398,7 @@ def get(external_request_id):
-                                                                 {}).get(dep.get('name'), [])
-                else:
-                    dep["topic_list"] = []
-
-        # Populate sentiment score for recommended packages
-        if recommendations:
-            for recommendation in recommendations:
-                alternate = recommendation['alternate']
-                for pkg in alternate:
-                    if reco_pkg_sentiment_result is not None:
-                        reco_pkg_sentiment_item = \
-                            get_item_from_list_by_key_value(reco_pkg_sentiment_result.get(
-                                                                'task_result', {}).get(
-                                                                    'sentiment', []),
-                                                            'manifest_file_path',
-                                                            recommendation.get(
-                                                                'manifest_file_path'))
-                        pkg['sentiment']['overall_score'] = \
-                            reco_pkg_sentiment_item.get(pkg['name'], {}).get('score', 0)
-                        pkg['sentiment']['magnitude'] = \
-                            reco_pkg_sentiment_item.get(pkg['name'], {}).get('magnitude', 0)
-                    else:
-                        pkg['sentiment'] = {
-                            "latest_comment": "",
-                            "overall_score": 0,
-                            "magnitude": 0
-                        }
-
-                companion = recommendation['companion']
-                for pkg in companion:
-                    if reco_pkg_sentiment_result is not None:
-                        reco_pkg_sentiment_item = get_item_from_list_by_key_value(
-                            reco_pkg_sentiment_result.get('task_result', {}).get(
-                                'sentiment', []), 'manifest_file_path', recommendation.get(
-                                    'manifest_file_path'))
-                        pkg['sentiment']['overall_score'] = reco_pkg_sentiment_item.get(
-                            pkg['name'], {}).get('score', 0)
-                        pkg['sentiment']['magnitude'] = reco_pkg_sentiment_item.get(
-                            pkg['name'], {}).get('magnitude', 0)
-                    else:
-                        pkg['sentiment'] = {
-                            "latest_comment": "",
-                            "overall_score": 0,
-                            "magnitude": 0
-                        }
-
-        for stack in stacks:
-            stack["recommendation"] = get_item_from_list_by_key_value(
-                recommendations,
-                "manifest_file_path",
-                stack.get("manifest_file_path"))
+            stack["recommendation"] = get_item_from_list_by_key_value(recommendations, "manifest_file_path", stack.get("manifest_file_path"))
```

Comment:
The line is too long. Keep it as it is now please.

Reply:
reverted to its previous style.